NachoSpigot-Server/pom.xml

```diff
@@ -14,8 +14,8 @@
         <minecraft_version>1_8_R3</minecraft_version>
         <buildtag.prefix>git-Bukkit-</buildtag.prefix>
         <buildtag.suffix/>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
```

Comment:
We shouldn't do this and it drops support for all java versions under 17.

Reply:
fair enough, i'll go ahead and revert that commit